ovc5/scripts/device_scripts/init_usb_ethernet.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+service isc-dhcp-server stop
+ifconfig usb0 down
+
+killall dhcpd
+rm -f /var/run/dhcpd.pid
+rm -f /var/lib/dhcp/dhcpd.leases
+
+ifconfig usb0 10.0.1.1 netmask 255.255.255.0 up
+ifconfig usb0 mtu 13500
+
+ifconfig usb0 up
+service isc-dhcp-server start
```

Comment:
This is failing for me with the following error:

```
Apr 09 07:46:42 zynq dhcpd[1054]: Wrote 0 NA, 0 TA, 0 PD leases to lease file.
Apr 09 07:46:43 zynq dhcpd[1054]: 
Apr 09 07:46:43 zynq dhcpd[1054]: No subnet6 declaration for usb0 (no IPv6 addresses).
Apr 09 07:46:43 zynq dhcpd[1054]: ** Ignoring requests on usb0.  If this is not what
Apr 09 07:46:43 zynq dhcpd[1054]:    you want, please write a subnet6 declaration
Apr 09 07:46:43 zynq dhcpd[1054]:    in your dhcpd.conf file for the network segment
Apr 09 07:46:43 zynq dhcpd[1054]:    to which interface usb0 is attached. **
Apr 09 07:46:43 zynq dhcpd[1054]: 
Apr 09 07:46:43 zynq dhcpd[1054]: 
Apr 09 07:46:43 zynq dhcpd[1054]: No subnet6 declaration for eth0 (fe80::22b0:f7ff:fe04:f3a2).
Apr 09 07:46:43 zynq dhcpd[1054]: ** Ignoring requests on eth0.  If this is not what
Apr 09 07:46:43 zynq dhcpd[1054]:    you want, please write a subnet6 declaration
Apr 09 07:46:43 zynq dhcpd[1054]:    in your dhcpd.conf file for the network segment
Apr 09 07:46:43 zynq dhcpd[1054]:    to which interface eth0 is attached. **
Apr 09 07:46:43 zynq dhcpd[1054]: 
Apr 09 07:46:43 zynq dhcpd[1054]: 
Apr 09 07:46:43 zynq dhcpd[1054]: Not configured to listen on any interfaces!
Apr 09 07:46:43 zynq dhcpd[1054]: 
Apr 09 07:46:43 zynq dhcpd[1054]: If you think you have received this message due to a bug rather
Apr 09 07:46:43 zynq dhcpd[1054]: than a configuration issue please read the section on submitting
Apr 09 07:46:43 zynq dhcpd[1054]: bugs on either our web page at www.isc.org or in the README file
Apr 09 07:46:43 zynq dhcpd[1054]: before submitting a bug.  These pages explain the proper
Apr 09 07:46:43 zynq dhcpd[1054]: process and the information we find helpful for debugging.
Apr 09 07:46:43 zynq dhcpd[1054]: 
Apr 09 07:46:43 zynq dhcpd[1054]: exiting.
Apr 09 07:46:45 zynq isc-dhcp-server[1036]: Starting ISC DHCPv6 server: dhcpd6check syslog for diagnostic
Apr 09 07:46:45 zynq isc-dhcp-server[1036]:  failed!
Apr 09 07:46:45 zynq systemd[1]: isc-dhcp-server.service: Control process exited, code=exited, status=1/F
-- Subject: Unit process exited
-- Defined-By: systemd
-- Support: https://www.debian.org/support
```

I think you might have to add the interface name to `/etc/default/isc-dhcp-server`, for example:

```
INTERFACESv4="usb0"
```

I'm also not getting any USB enumeration happening on my host machine, did this happen to you?

Reply:
Oh I think I must've done this and forgot to add it to the script. I'll get on it.